config/convert2rhel.ini

```diff
@@ -11,3 +11,14 @@
 # password       = <insert_password>
 # activation_key = <insert_activation_key>
 # org            = <insert_org>
+
+[host_metering]
+# configure_host_metering          = "0"
```

Comment:
Just a question, but shouldn't the host metering value be either "auto" or "force"?

Reply:
Hmmm. That's true, not sure why I changed to 0 or 1... Let me fix that

Reply:
Thanks!